tests/unit/test_mock_build.py

```diff
@@ -0,0 +1,6 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+def test_build_in_mock_default_resultdir():
```

Comment:
You're definitely missing a directory with sources for this… You can fix this by adding a fixture to the test parameters here:
```suggestion
def test_build_in_mock_default_resultdir(upstream_and_remote):
    cwd, _ = upstream_and_remote
```